File: src/main/csharp/p00/Problem0006.cs
namespace ProjectEuler.main.csharp.p00
{
    public class Problem0006
    {
        public static int Solve()
        {
            var sum1 = 0;
            var sum2 = 0;

            for (var i = 1; i <= 100; i++)
            {
                sum1 += i * i;
                sum2 += i;
            }

            return sum2 * sum2 - sum1;
        }
    }
}

File: src/main/kotlin/p00/Problem0006.kt
package p00

class Problem0006 {

    fun solve(): Int {
        var sotsquares = 0
        var sotsum = 0

        for (i in 1..100) {
            sotsquares += i * i
            sotsum += i
        }

        return sotsum * sotsum - sotsquares
    }

}